docs/UserGuide.md

```diff
@@ -401,20 +401,36 @@ Parameters must follow the prefix they are associated with and are typically in
 `add n/NAME p/PHONE i/INCOME e/EMAIL a/ADDRESS f/FAMILY t/TAG h/HOUSING_TYPE [r/REMARK] [b/BIRTHDAY]`
 </box>
 
-<box type="info" header="Notes">
```

Comment:
I think Features section should keep/add and reference to Field Constraints

![image](https://github.com/AY2324S2-CS2103T-W10-1/tp/assets/39399745/8c00339a-ea7f-4ed8-a5ed-5947f1580973)

Some of the points should be separated, 
`PREFIX/ tags are used to specify the type of data being entered. For example, in add n/John Doe p/98765432, n/ stands for name, and p/ stands for phone number. Like command words, prefixes are also case-insensitive (n/ and N/ are treated the same).`
one point for what is prefix and another for case insensitive, they are two features all together